tsconfig.json

```diff
@@ -18,6 +18,7 @@
     // Workaround for https://github.com/microsoft/TypeScript/issues/55132
     "useDefineForClassFields": false,
     "allowImportingTsExtensions": true,
+    "skipLibCheck": true,
```

Comment:
This seems to be a step back in how strict we do our typing.
Are we sure we want this? Is there no way to fix the reason we add this with yarns resolutions?
Can it get a comment why we need this now?